python/social_bot/models/youbot_noplugin/model.sdf

```diff
@@ -711,24 +703,6 @@
       <self_collide>0</self_collide>
       <kinematic>0</kinematic>
     </link>
-    <link name="caster_link_br">
```

Comment:
Here's explanations for these changes in model file:
Similar to PR2, the Youbot model have a "fake" Mecanum wheel and use a caster joint instead for turning (does not have the free DoF for omni sub wheel because it's hard to simulate).  By removing the caster link and joints and use less friction for back wheel link this works similar to differential drive, which is both easier to control and decrease the computational cost for simulation.

Reply:
Please add this as a comment to the model file, so it's documented.